source/ranges.tex

```diff
@@ -5967,6 +5992,9 @@
     @\exposid{inner-iterator}@() = default;
     constexpr explicit @\exposid{inner-iterator}@(@\exposid{outer-iterator}@<Const> i);
 
+    constexpr const iterator_t<@\exposid{Base}@>& base() const &;
```

Comment:
@jensmaurer, @timsong-cpp, @BRevzin: the requires clause (`requires copyable<iterator_t<Base>>`) is missing here, is that intentional?

Reply:
I'm adding the requires clause, as presented in the paper.

Reply:
Yes, the missing requires-clause is intentional. The return type was changed to "const blah &", so no copy is performed.

Reply:
See LWG3533 Make base() const & consistent across iterator wrappers that supports input_iterators
in commit 92fe70a3fa75b3abad8c5f06fdadf3990aaf4338.

Reply:
Yep - see LWG3533

Reply:
Yes, right, that was in the previously separate commit. Got it, thanks!

Reply:
That's the REASON why the commit was separate: It was not supposed to be squashed, because it was an sort-of editorial fix beyond the paper on the plate.

Reply:
Yeah, it makes sense. I had several other merge conflicts to resolve so I wasn't sure at the time. I'll separate this one out again.